### PR TITLE
Replaces status frame sprite.

### PR DESCRIPTION
**Pull request by Belsima**
images
Old left, new right.

Taken from CEV-Eris and a random folder on my desktop. I mashed the sprites together. Also fixes a minor bug with pens because why not.

### CODE REVIEW

**Comment by Anewbe** [comment]
Don't mix and match PRs, please. One PR, one change is sort of the guideline.

**Comment by Belsima** [comment]
It was honestly an accident because I was doing multiple things at once and pushed the pen file by accident, but it's a 1 line bugfix so I figured it wasn't worth undoing.

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -79,7 +79,6 @@
 
 /obj/item/weapon/pen/reagent
 	flags = OPENCONTAINER
-	slot_flags = SLOT_BELT
```
Comment (SpadesNeil) [inline code/modules/paperwork/pen.dm]: Why'd this get removed? I know it's accidental but still.

Comment (Belsima) [inline code/modules/paperwork/pen.dm]: So that they can go into the ear slot, like regular pens.

Comment (SpadesNeil) [inline code/modules/paperwork/pen.dm]: Oh ok.

**Pull request merged**